python/pdo/contract/state.py

```diff
@@ -186,6 +188,7 @@ def __pull_blocks_from_eservice__(cls, eservice, contract_id, block_ids, data_di
         block_count = len(blocks_to_pull)
         if block_count == 0 :
             logger.debug('no blocks to pull')
+            logger.info("Pulled 0 new blocks after contract update")
```

Comment:
no reason to have two consecutive logger statements

Reply:
thank you, removed the info log